Microsoft.Maui.WinUI.sln

```diff
@@ -36,6 +36,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Maui.Controls.Sample.WinUI"
 EndProject
 Project("{C7167F0D-BC9F-4E6E-AFE1-012C56B48DB5}") = "Maui.Controls.Sample.WinUI (Package)", "src\Controls\samples\Controls.Sample.WinUI (Package)\Maui.Controls.Sample.WinUI (Package).wapproj", "{503B7824-7376-450B-9EF7-EA587DF2C8BE}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "BlazorWebView", "BlazorWebView", "{437D574C-FB97-4967-B521-8B8E66E93F69}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Components.WebView.Maui", "src\BlazorWebView\src\Microsoft.AspNetCore.Components.WebView.Maui\Microsoft.AspNetCore.Components.WebView.Maui.csproj", "{0F52573B-3F52-4064-9DBA-B601FB14F559}"
```

Comment:
Will the 
`src\BlazorWebView\`

Folder gain samples and tests?

Will the 
`src\BlazorWebView\src`

Folder gain additional projects?

A couple thoughts to reduce folder length
- just call the  `Microsoft.AspNetCore.Components.WebView.Maui`  folder `core`. That's what we've done for the other libraries to reduce folder lenght
- Perhaps just get rid of that folder and put this all into `src\BlazorWebView\src`

Reply:
Tests? Yeah... 😁 I'll simplify these names to be shorter.